src/types/statsig.ts

```diff
@@ -0,0 +1,5 @@
+export type StatsigConfigType = Record<StatSigFlags, boolean>;
```

Comment:
nit, I wonder if to follow existing directory structure better, we should move this into `src/constants/statsig.tsx`?

Reply:
ah so this is on purpose. i can revert this change but it's often considered best practice to separate constants and types, it will help prevent cyclic dependencies

> this pr also creates a types folder that will allow us to separate constant values from types. which will help us prevent cyclic dependencies

Reply:
oh my b - didn't read that, sgtm